app/src/main/res/layout/onboarding_fragment.xml

```diff
@@ -66,6 +66,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginEnd="20dp"
+        android:contentDescription="@string/next_arrow"
```

Comment:
There are four variations of `onboarding_fragment` file.
`layout/onboarding_fragment`, `layout-land/onboarding_fragment`, `layout-sw600dp-land/onboarding_fragment` and `layout-sw600dp-port/onboarding_fragment`

This change is done only in `layout/onboarding_fragment`. Please apply this change in other files too.

Reply:
> @adit19shah Suggested changes.
> 
> In general focus on minute details like code consistency, patterns, etc.

Got it...will be careful of these things next time.

Reply:
> There are four variations of `onboarding_fragment` file.
> `layout/onboarding_fragment`, `layout-land/onboarding_fragment`, `layout-sw600dp-land/onboarding_fragment` and `layout-sw600dp-port/onboarding_fragment`
> 
> This change is done only in `layout/onboarding_fragment`. Please apply this change in other files too.

Done.